Ginger/Ginger/Actions/ActionEditPages/ActWebSmartSyncEditPage.xaml.cs

```diff
@@ -89,69 +89,69 @@ private void NotifyLocatorVisibilityChanged()
                     break;
-                case ActNewSmartSync.eSyncOperation.EnabilityOfAllElementsLocatedBy:
+                case ActWebSmartSync.eSyncOperation.EnabilityOfAllElementsLocatedBy:
                     xOperationDescription.Text = "Checks if all the elements found by a given method are enabled and can be interacted. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.FrameToBeAvailableAndSwitchToIt:
+                case ActWebSmartSync.eSyncOperation.FrameToBeAvailableAndSwitchToIt:
                     xOperationDescription.Text = "Waits until a frame is available to switch to and then switches to it. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.InvisibilityOfAllElementsLocatedBy:
+                case ActWebSmartSync.eSyncOperation.InvisibilityOfAllElementsLocatedBy:
                     xOperationDescription.Text = "Waits until all the elements found by a given method are invisible or not present. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.InvisibilityOfElementLocated:
+                case ActWebSmartSync.eSyncOperation.InvisibilityOfElementLocated:
                     xOperationDescription.Text = "Waits until a specific element is no longer visible or not present. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.PresenceOfAllElementsLocatedBy:
+                case ActWebSmartSync.eSyncOperation.PresenceOfAllElementsLocatedBy:
                     xOperationDescription.Text = "Ensures that all the elements found by a given method are present in the webpage's structure (DOM). ";
                     break;
-                case ActNewSmartSync.eSyncOperation.SelectedOfAllElementsLocatedBy:
-                    xOperationDescription.Text = "Ensures that all the elements found by a given method are visible on the webpage. ";
+                case ActWebSmartSync.eSyncOperation.SelectedOfAllElementsLocatedBy:
+                    xOperationDescription.Text = "Ensures that all the elements found by a given method are selected. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.UrlMatches:
+                case ActWebSmartSync.eSyncOperation.UrlMatches:
                     xOperationDescription.Text = "Waits until the URL of the current page matches a specified pattern. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.VisibilityOfAllElementsLocatedBy:
-                    xOperationDescription.Text = "Ensures that all the elements found by a given method are selected. ";
+                case ActWebSmartSync.eSyncOperation.VisibilityOfAllElementsLocatedBy:
+                    xOperationDescription.Text = "Ensures that all the elements found by a given method are visible on the webpage. ";
```

Comment:
change it to "Ensures that all the elements found by a given locator are visible on the webpage.